CHANGELOG.md

```diff
@@ -4,14 +4,120 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.18.0] - XXX. XX, 2024
+## [0.18.0] - Sept. XX, 2024
+
+This release reaches an important milestone by making offloading fully asynchronous.
+Calls to `dpctl.tensor` submit tasks for execution to DPC++ runtime and return without waiting for execution of these tasks to finish.
+The sequential semantics a user comes to expect from execution of Python script is preserved though.
+
+The full list of changes that went into this release are:
 
 ### Added
 
+* Implement `tensor.take_along_axis` per Python Array API specification [gh-1778](https://github.com/IntelPython/dpctl/pull/1778)
+* Implement `tensor.put_along_axis` to complement `tensor.take_along_axis` [gh-1798](https://github.com/IntelPython/dpctl/pull/1798)
+* Support for 'device=tensor.kDLCPU' in `tensor.from_dlpack` function and `tensor.usm_ndarray.__dlpack__` method [gh-1781](https://github.com/IntelPython/dpctl/pull/1781)
+* Support DLPack on Windows [gh-1746](https://github.com/IntelPython/dpctl/pull/1746)
+* Implement `tensor.nextafter` function per Python Array API specification [gh-1730](https://github.com/IntelPython/dpctl/pull/1730)
+* Implement `tensor.count_nonzero` and `tensor.diff` functions from Python array API specification [gh-1732](https://github.com/IntelPython/dpctl/pull/1732), [gh-1780](https://github.com/IntelPython/dpctl/pull/1780)
+* Add support for `order="K"` to `*_like` array creation functions, and change default `order` keyword value from `'C'` to `'K'` [gh-1808](https://github.com/IntelPython/dpctl/pull/1808)
+* Support for 'max dimensions' in Array API capabilities info data [gh-1774](https://github.com/IntelPython/dpctl/pull/1774)
+* Add support for device aspect 'simulated' [gh-1691](https://github.com/IntelPython/dpctl/pull/1691)
```

Comment:
Wasn't the device aspect `emulate` not simulated?

Reply:
Yes, indeed. Good catch!